app/policies/dashboard/booking_policy.rb

```diff
@@ -0,0 +1,13 @@
+module Dashboard
+class BookingPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+    def show?
+      true
+    end
+end
+
+end
```

Comment:
Same problem as booking_policy outside the dashboard folder. The "end" is in the wrong places. Now the "show?" method is outside Scope class.